morphocut/str.py

```diff
@@ -10,6 +10,9 @@
 class Format(Node):
     """
     Format a string just like :py:meth:`str.format`.
+
+    This class allows you to create and customize your own string formatting behaviors using the same 
+    implementation as the :py:meth:`str.format` methods.
```

Comment:
``... using the :ref:`Python Format String Syntax <python:formatstrings>`.``

I explicitly asked you to include a link to ``:ref:`python:formatstrings` ``.

Reply:
Kindly check if it is ok now?